applicant/templates/applicant/listing_detail.html

```diff
@@ -6,28 +6,35 @@
 
 
 {% block content %}
-  <h1>SalarySleuth</h1>
-  <a href="{% url 'applicant:listings' %}">
-    <button>Back to listings</button>
-</a>
-
-  <h2>Company</h2>
-  <h3>Offer - Location</h3>
-
-  <a href="{% url 'applicant:profile' uid=1 %}">
-    <button >Profile</button>
-    </a>
-    <a href="{% url 'applicant:choose_info' uid=1 lid=1%}">
-    <button >Apply</button>
+    <div>
+        <h1>SalarySleuth</h1>
+    </div>
+
+    {% block header %}
+            <a class="or_button" href="{% url 'applicant:profile' uid=1 %}">
+                Profile
+            </a>
+    {% endblock %}
+
+    <a class="or_button" href="{% url 'applicant:listings' %}">
+        Back to listings
     </a>
 
-<h2>Job Description</h2>
-<h3>aaaaaaaaaaaaaaaaaaaaaaaaaaaa
-    aaaaaaaaaaaaaaaaaaaaaaaaaaaaa
-    aaaaaaaaaaaaa
-</h3>
+    <div>
+        <h2>Company</h2>
+        <h3>Offer - Location</h3>
+    </div>
 
+    <div>
+    
+        <a class="or_button" href="{% url 'applicant:choose_info' uid=1 lid=1%}">
+            Apply
+        </a>
+    </div>
 
+    <div>
+        <h2>Job Description</h2>
+        <h3>aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa</h3>
```

Comment:
nei

Reply:
im literally working on this page rn
u interrupted me to **FORCE** me to update

Reply:
fine

Reply:
for now...

Reply:
ekki gleyma að taka þetta